sites/geohub/src/routes/(app)/data/upload/+page.svelte

```diff
@@ -119,6 +131,19 @@
 								});
 							})
 						);
+					} else {
+						zipEntryPromises.push(
+							zipEntry.async('blob').then((blob) => {
+								zipFileList.push({
+									name: zipEntry.name,
+									path: zipFile.path
+										? zipEntry.name
+										: `${zipFile.name.replace('.zip', '')}/${zipEntry.name}`,
```

Comment:
probably you need to check other archived formats as well (.7z, .tar, etc)?

Reply:
@JinIgarashi 
1. If the user selects single file shapefiles example `sdr_subnational.shp`, `sdr_subnational.prj` they are immediately zipped using `JsZip` into a file named `sdr_subnational.zip` which is displayed on the table. Any corresponding shapefile files that are selected are added to this zipfile
2. The highlighted code is used inside the `getZipFilesList` which is in turn used in the `checkShapeFileValidity` function which returns a mapping in the format `{"shapefile_name":["list","of","missing","shapefiles"]}`
3. The `checkShapefile` validity checks for shapefile validity in these zip files, _**not the ones added as zip files (or any other format) directly by the user.**_. 
4. We don't zip the shapefiles into any other archive format. **_Only_** `.zip` as currently there are no more checks for archive files uploaded by the user, but rather only the shapefiles that have been zipped by GeoHub. Only shapefiles that have been uploaded by the user as single files are zipped